### PR TITLE
Typos in README

### PR DESCRIPTION
**Pull request by elevran**

### CODE REVIEW

**Comment by elevran** [comment]
corporate CLA (IBM) in place

**Comment by elevran** [comment]
CLA signed - dear bot PTAL

**Comment by elevran** [comment]
@JeremyOT would it be possible to get a quick 5m review for this?

File: keps/sig-multicluster/1645-multi-cluster-services-api/README.md [hunk 1]
```diff
@@ -234,7 +233,7 @@ you're proposing, but should not include things like API designs or
 implementation.  The "Design Details" section below is for the real
 nitty-gritty.
 -->
-#### Terminology
+### Terminology
```
Comment (JeremyOT) [inline keps/sig-multicluster/1645-multi-cluster-services-api/README.md:236]: I'd rather leave this as it was - terminology is part of the preamble and doesn't need the same weight as the other sections.

**Review by JeremyOT — COMMENTED** [review]
Thanks for fixing these. Do you mind squashing your commits?

File: keps/sig-multicluster/1645-multi-cluster-services-api/README.md (hunk 1 above)
Comment (elevran) [reply keps/sig-multicluster/1645-multi-cluster-services-api/README.md]: done: reverted level and squashed commits.
Thanks for the quick turnaround!

**Comment by elevran** [comment]
@JeremyOT - fixed pre review comments and squashed. PTAL

**Comment by elevran** [comment]
@JeremyOT - PTAL. All fixed per your request.

**Comment by elevran** [comment]
@LappleApple - is this PR still being considered or should I close it?

**Comment by LappleApple** [comment]
Hi @elevran: Just saw this now. Thank you for the PR. 

/lgtm

**Comment by elevran** [comment]
thanks for the quick response @LappleApple! Was assuming this was being handled due to being moved to "In progress in Enhancements Subproject" earlier in Nov, but then it was removed so was wondering... Guess it only needs the `approved` label now.

**Comment by JeremyOT** [comment]
Thanks @elevran - sorry for the wait, this fell off my radar

/lgtm
/approve

**Pull request merged**